components/dashboard/src/admin/TeamDetail.tsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import moment from "moment";
+import { useEffect, useState } from "react";
+import { Team, TeamMemberInfo } from "@gitpod/gitpod-protocol";
+import { getGitpodService } from "../service/service";
+import { Item, ItemField, ItemsList } from "../components/ItemsList";
+
+ export default function TeamDetail(props: { team: Team }) {
+    const { team } = props;
+    const [teamMembers, setTeamMembers] = useState<TeamMemberInfo[] | undefined>(undefined);
+
+    useEffect(() => {
+        (async () => {
+            const members = await getGitpodService().server.adminGetTeamMembers(team.id);
+            if (members.length > 0) {
+                setTeamMembers(members)
+            }
+        })();
+    }, [team]);
+
+     return <>
+        <div className="flex">
+            <div className="flex-1">
+                <div className="flex"><h3>{team.name}</h3><span className="my-auto">{team.markedDeleted ? " Deleted" : ""}</span></div>
+                <p>{teamMembers && teamMembers.length} member(s)</p>
+                <p>Created {moment(team.creationTime).format('MMM D, YYYY')}</p>
```

Comment:
**question**: Should we align the creation timestamp information structure on a) teams list and the b) team page? For example, we show _Created Feb 23, 2022_ and _a few seconds ago_ for the same entry. Happy to move forward with this small inconsistency. In the first case, let's add an adverb, like _Created on Feb 23, 2022_. What do you think?

Reply:
Oops good catch, thanks! Will fix the one on the search result to also be a specific date, and add "on".

Reply:
@laushinka Long-term I'd opt for going with the other option to avoid duplication of information (_Created on ..._) as this is also repeated on the table header. This is what we've practiced so far in other areas of the dashboard. You call!

Reply:
I talked about this earlier with @jldec and opted for the specific date because it would be more useful for the admin to know when a team was created. On other cases I can see where we would like to avoid duplication, but for debugging purposes the duplication would help for both when navigating the teams search result and visiting the team detail 🙏🏽

Reply:
DEAL. Fair point. 🤝